File: demo/src/main/java/com/example/demo/BootReceiver.java
package com.example.demo;


import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;
import android.util.Log;

public class BootReceiver extends BroadcastReceiver {

    private final String ACTION_BOOT = "android.intent.action.BOOT_COMPLETED";
    @Override
    public void onReceive(Context context, Intent intent) {
        if(intent.getAction().equals(ACTION_BOOT)){
            Log.d("yuanwei", "com.seuic.kysy isAvailable" + "ACTION_BOOT mAppRow.systemApp = ");
        }

    }
}

File: kotlin/src/main/java/com/yw/kotlin/BootReceiver.kt
package com.yw.kotlin

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import android.widget.Toast

class BootReceiver : BroadcastReceiver() {
    private val actionBoot: String = "android.intent.action.BOOT_COMPLETED"

    override fun onReceive(context: Context?, intent: Intent?) {
        if (intent?.action.equals(actionBoot)) {
            Toast.makeText(context, "Kotlin demo 开机广播接收到啦!", Toast.LENGTH_LONG).show()
        }
    }
}